packages/manager/modules/log-to-customer/src/constants.js

```diff
@@ -0,0 +1,49 @@
+export const LOGS_INFO = {
+  FR: 'https://docs.ovh.com/fr/logs-data-platform/kubernetes-fluent-bit/',
+  DEFAULT:
+    'https://docs.ovh.com/gb/en/logs-data-platform/kubernetes-fluent-bit/',
+};
```

Comment:
You should mark all data which will be parameterized later with a TODO to identify them more easily